src/abandoned-cart-email-widget/abandoned-cart-email-widget.html

```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+
+The Sezzle Shopify abandoned cart email widget is a lightweight version of our static widget.It suggests the merchants to use Sezzle widget in their brand’s email footers and abandoned cart emails.
```

Comment:
This doesn't necessarily need to be exclusive to Shopify though. Perhaps we can tailor the language to say the below instructions were written for Shopify but the code can be modified for other platforms as needed?